### PR TITLE
chore: disable pylint running on main branch

### PR DESCRIPTION
**Pull request by aarnphm**
Signed-off-by: Aaron Pham <29749331+aarnphm@users.noreply.github.com>

## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -2,7 +2,6 @@ include README.md
 include LICENSE
 
 # Included in PyPI distribution
-include bentoml/*.py
 include bentoml/_version.py
```
Comment (parano) [inline MANIFEST.in:5]: all these lines can be removed

Comment (parano) [reply MANIFEST.in:5]: only `graft bentoml` is needed

Comment (parano) [reply MANIFEST.in]: actually that's not needed neither, just suggested a change fromt he UI

Comment (aarnphm) [reply MANIFEST.in]: noted

**Pull request merged**